packages/app-mobile/root.tsx

```diff
@@ -618,6 +618,12 @@ class AppComponent extends React.Component {
 		this.onAppStateChange_ = () => {
 			PoorManIntervals.update();
 		};
+
+		this.handleOpenURL_ = (event: any) => {
+			if (event.url == ShareExtension.shareURL) {
```

Comment:
As I understand ShareExtension.shareURL will be an empty string in Android, but how about event.url? Is it also an empty string? Just trying to understand how it will work in Android.

Reply:
I do not think this will be ever called on Android unless there's a corresponding change in manifest.
If I read it correctly, this listener will be called if user open a url like  `url://<....>`

Reply:
Yes that's correct. For the iOS app, I added a `joplin://` URI scheme which launches the main app from the share extension. Nothing like this was added for Android, so `handleOpenURL` shouldn't be called.